### ISSUE TITLE
Tensorflow 2.5.0 not working on Colab

### ISSUE THREAD

**Issue by RLC2000** [googlecolab/colabtools#3663]
I had made a ML project on Masked RCNN in google colab that was working fine on March 25, 2023 but when I tryed to re-run the same notebook on 9 May, 2023 I'm getting a error in cell 2.
Here is the error log:
`
ERROR: Could not find a version that satisfies the requirement tensorflow==2.5.0 (from versions: 2.8.0rc0, 2.8.0rc1, 2.8.0, 2.8.1, 2.8.2, 2.8.3, 2.8.4, 2.9.0rc0, 2.9.0rc1, 2.9.0rc2, 2.9.0, 2.9.1, 2.9.2, 2.9.3, 2.10.0rc0, 2.10.0rc1, 2.10.0rc2, 2.10.0rc3, 2.10.0, 2.10.1, 2.11.0rc0, 2.11.0rc1, 2.11.0rc2, 2.11.0, 2.11.1, 2.12.0rc0, 2.12.0rc1, 2.12.0)
ERROR: No matching distribution found for tensorflow==2.5.0
`
Behavior on 25 March, 2023 (Expected Behaviour)
`
Collecting tensorflow==2.5.0
		  Downloading tensorflow-2.5.0-cp38-cp38-manylinux2010_x86_64.whl (454.4 MB)
		[2K     [90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━[0m [32m454.4/454.4 MB[0m [31m3.8 MB/s[0m eta [36m0:00:00[0m
		[?25hCollecting keras-nightly~=2.5.0.dev
		  Downloading keras_nightly-2.5.0.dev2021032900-py2.py3-none-any.whl (1.2 MB)
		[2K     [90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━[0m [32m1.2/1.2 MB[0m [31m75.1 MB/s[0m eta [36m0:00:00[0m
.
.
.		
`
The browser that I am using is Brave [Version 1.51.114 Chromium: 113.0.5672.92 (Official Build) (64-bit)]

Here is a link to my Notebook: [https://colab.research.google.com/drive/1hHV4U8sCLxnQ_jteG93EcBiOE2UUCrcs?usp=sharing](url)

**Comment by EvanWiederspan** [comment]
Tensorflow is now on 2.12, 2.5 is very old and does not seem to be installable on Python 3.10. I would recommend updating your notebook to use a newer version of Tensorflow, I believe 2.8 is the oldest that we now support

**Comment by RLC2000** [comment]
Hi Evan Thanks for replying , Is there a work around something using with we can install tf 2.5 in our colab. we are actually students and are not that well versed with ML we have done the project after referring multiple videos on MaskRCCN and almost all of them are using tf 1.X version or Tf 2.Y where Y<=5 version.

we had made a copy of the notebook and installed tf 2.8 on it but after that some packages stoppped working like  scikit-image 0.16.1 and we get some new error to which there is no stackoverflow thread too.

we just need to run the model once on our actually dataset and download the model for future use. Is there someting that you can suggest, please. Anything even something on best effort basis will help.
Thank you.

**Comment by EvanWiederspan** [comment]
If you just need to run this once, you may still be able to use the Fallback version from the command palette for this. If that still doesn't work, then you will just have to update the notebook a bit to use it on colab. You can try using a newer scikit-image version, we install 0.19 by default right now
